### DISCUSSION TITLE
i added new store but i get error

### DISCUSSION THREAD

**Discussion by razor992fg** [jef/streetmerchant#1951] [Help]
Hello guys, i tried to add a new store on streetmerchant but i get error when i try to start bot.

Anyone can check if there are errors?

```
import {Store} from './store';

export const mediaworld: Store = {
	labels: {
		inStock: [
			{
				container:
					'product-add-to-cart.mw-yellow-btn.btn-size-md.js-add-to-cart',
				text: ['Aggiungi al carrello']
			}
		],
		maxPrice: {
			container: '#normalPriceAmount',
			euroFormat: false
		},
		outOfStock: {
			container: 'search-product-widget-title.red-highlight',
			text: ['Nessun risultato trovato']
		}
	},
	links: [
		{
			brand: 'sony',
			model: 'ps5 console',
			series: 'sonyps5c',
			url:
				'https://www.mediaworld.it/catalogo/console-e-pc-gaming/sony-playstation-5/f/prezzo-400-500-500'
		},
		{
			brand: 'sony',
			model: 'ps5 digital',
			series: 'sonyps5de',
			url:
				'https://www.mediaworld.it/catalogo/console-e-pc-gaming/sony-playstation-5/f/prezzo-300-400-400'
		},
	],
	name: 'mediaworld'
};
```

**Comment by OptimalMising** [comment]
What error do you get?

Your store model is likely fine, have you edited the index.ts file to include your new store as shown here:
https://jef.codes/streetmerchant/help/general/#updating-the-models

**Reply by razor992fg** [reply]
Yes. Streetmerchant not start

**Reply by ercobre** [reply]
@razor992fg have you solved this problem ?, i'm looking for mediaworld too but get same error.
ce la facciamo a prendere sta play 5 ? :D

**Reply by Justsnooze** [reply]
Resolved?

**Reply by ercobre** [reply]
Nope

**Comment by unekgal** [comment]
Steps to add new store or new brand/model/series.

1. Copy/paste an existing store, edit with new store info, save as new store name
2. Add brand, model, and series to store.ts
3. add the new store name to 2 locations in index.ts
4. Add DISCORD_NOTIFY_GROUP and MAX_PRICE_SERIES series to config.ts
6. Add DISCORD_NOTIFY and MAX_PRICE series in .env

**Comment by ghost** [comment]
Could you inlcude the error you're receiving?
